tomcat/src/main/java/camp/nextstep/handler/AbstractController.java

```diff
@@ -4,16 +4,18 @@
 import org.apache.coyote.http11.model.HttpRequest;
 import org.apache.coyote.http11.model.HttpResponse;
 
+import java.io.IOException;
+
 public abstract class AbstractController implements RequestHandler {
 
     @Override
-    public void handle(HttpRequest request, HttpResponse response) throws Exception {
+    public void handle(HttpRequest request, HttpResponse response) throws IOException {
         switch(request.getHttpMethod()) {
             case GET -> doGet(request, response);
             case POST -> doPost(request, response);
         }
     }
 
-    protected void doPost(HttpRequest request, HttpResponse response) throws Exception { /* NOOP */ }
-    protected void doGet(HttpRequest request, HttpResponse response) throws Exception { /* NOOP */ }
+    protected void doPost(HttpRequest request, HttpResponse response) throws IOException { /* NOOP */ }
+    protected void doGet(HttpRequest request, HttpResponse response) throws IOException { /* NOOP */ }
```

Comment:
```suggestion
    protected abstract void doPost(HttpRequest request, HttpResponse response) throws IOException;
    protected abstract void doGet(HttpRequest request, HttpResponse response) throws IOException;
```

Exception 처리 상세하게 해주셔서 좋습니다. 👍 
자식 클래스에서 override 해야할 메서드는 abstract 문법을 사용해주시면 어떨까 제안해봅니다.

Reply:
앗 좋네요!!